diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/comments/HeaderCommentRule.kt

```diff
@@ -190,6 +190,34 @@ class HeaderCommentRule(private val configRules: List<RulesConfig>) : Rule("head
         }
     }
 
+    /**
+     * Deletes all spaces and newlines
+     * Used to compare copyrights in yaml and file
+     */
+    private fun String.flatten(): String =
+            replace("\n", "")
+                .replace(" ", "")
+
+    /**
+     * If it is multiline copyright, this method deletes spaces in empty lines.
+     * Otherwise, if it is one line copyright, it returns it with 4 spaces at the beginning.
+     */
+    private fun handleMultilineCopyright(copyrightText: String): String {
+        if (copyrightText.contains("\n")) {
```

Comment:
I think it's better to check if the string start with space here, because single line strings can have explicit indentation too.